LICENSE: Add MIT LICENSE file

ayan-b opened the pull request:
coala/gci-leaders uses MIT LICENSE. A file containing the
MIT LICENSE named LICENSE.txt is added.

Closes https://github.com/coala/gci-leaders/issues/6

jayvdb commented:
This doesnt render properly as markdown.
See https://github.com/coala/gci-leaders/blob/6081e7ce7e4a7f199643a4829d97576f6f9bbc48/LICENSE.md

Better to not try to create a markdown file.

Plain text is good enough.

ayan-b committed:
LICENSE: Add MIT LICENSE file

coala/gci-leaders uses MIT LICENSE. A file containing the
MIT LICENSE named LICENSE.txt is added.

Closes https://github.com/coala/gci-leaders/issues/6

jayvdb commented:
ack https://github.com/coala/gci-leaders/pull/155/commits/dd87329281ac86fbd45b3aaa8d600fe41f1b3094

jayvdb commented:
I am using this PR to test the release candidate of GitMate.  It seems to not be setup correctly, so we will keep using this PR to verify when it is working correctly.

Please do not modify this PR.

jayvdb commented:
ack dd87329

jayvdb commented:
@gitmate-bot ff

refeed commented:
@jayvdb Could you try to merge this PR again?
